cmd/webhook/main.go

```diff
@@ -116,6 +124,8 @@ func main() {
 	flag.IntVar(&opts.Port, "secure-port", opts.Port, "The port on which to serve HTTPS.")
 
 	flag.Parse()
+	resourcesNamesList := strings.Split(*resourcesNames, ",")
+    listResources = append(listResources, resourcesNamesList...)
```

Comment:
and here. Maybe run gofmt on these files?

Reply:
Same as the above comment the format at the editor looks fine.

Reply:
The editor might display things with spaces vs. tabs, but gofmt which ensures that regardless of which ide you use the code looks the same is not agreeing, for example:
https://github.com/sigstore/policy-controller/actions/runs/12412408356/job/34695099927?pr=1687

Did you try running gofmt? At the root if you do:
```
gofmt -s -w .
```

And then it should update everything for you.

Reply:
Done